ptbr/tests/test_config_cli.py

```
@@ -0,0 +1,836 @@
+            _coerce_type("not a list", list)
+
+
+# ============================================================================
+# Unit tests: _validate_section
+# ============================================================================
+
+
+class TestValidateSection:
+    def test_required_field_missing_errors(self):
+        report = ValidationReport()
+        result = _validate_section({}, _GLINER_RULES, "gliner_config", report)
```

Ruff (0.15.1)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)